components/module-components/Mission&Vision/missionVision.module.scss

```diff
@@ -139,7 +135,7 @@
 }
 
 .missionTitleDark {
-  background: rgba(54, 67, 98, 1);
+  background: var(--Primary, hsla(222, 50%, 14%, 1));
```

Comment:
could you update the colors file as well? (colors.scss) and add this values as well so we can reuse them through out the project